main.py

```diff
@@ -69,14 +69,28 @@ def generate_plot_benchmark(fname, kinds=PLOT_KINDS):
                 plot_func = globals()[PLOT_KINDS[k]]
                 try:
                     fig = plot_func(df_obj, plotly=True)
+                    if plot_func != "plot_histogram":
+                        if len(df_obj["solver_name"].unique()) < 10:
+                            fact_ = 10
+                        else:
+                            fact_ = 100
```

Comment:
I don't understand why the factor is changing this much?

Reply:
Because the more curves there are, the more the legend takes on the actual plot (it shrinks upwards).
This just creates enough space for a new line (the 10 could be 0 or anything very low technically).